packages/docusaurus-plugin-content-docs/src/__tests__/lastUpdate.test.ts

```diff
@@ -10,8 +10,7 @@ import fs from 'fs-extra';
 import path from 'path';
 import shell from 'shelljs';
 import {createTempRepo} from '@testing-utils/git';
-
-import {getFileLastUpdate} from '../lastUpdate';
+import {getFileLastUpdate} from '@docusaurus/utils';
```

Comment:
This is not testing the docs plugin anymore so this test should be moved to utils tests

Reply:
This file should also add unit tests for `readLastUpdateData`: it will be simpler to test this method rather than using blog plugin `loadContent`

And this code is shared between docs/blog so we don't want to test it twice inside each plugin using it.